includes/Admin/Customizer.php

```diff
@@ -124,6 +125,21 @@ public function register() {
 		add_action( 'customize_register', [ $this, 'register_customizer_settings' ] );
 	}
 
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @since 1.16.0
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed(): bool {
```

Comment:
Worth adding a test?

Reply:
What would it test? That is returns false?

Reply:
I was thinking of a small unit test, but also an e2e test that verifies that the customizer is properly hidden.

Reply:
How would I do that? I think that should be added once we test against 5.9? 
This change is simple and clear, no reason to block it.

Reply:
Create a follow up ticket to add a e2e test?

Reply:
> How would I do that?

Adding an e2e test would be pretty simple, no?

But yes, this needs to be done once we test against RC.

> This change is simple and clear, no reason to block it.

It's still a change that needs to be tested :-) Hence my original question whether it's worth adding tests.